server/controllers/shared/autoSaveController.ts

```diff
@@ -2,22 +2,27 @@ import { Request, Response } from 'express'
 
 import { IFieldValue } from '../../services/reportService'
 import SharedController from './sharedController'
+import Report from '../../repositories/entities/report'
 
 interface IPageField {
   id: string
   value: string
 }
 
 export default class AutoSaveController extends SharedController {
-  protected override updateFields = async (pageFields: Array<IPageField>) => {
+  protected override updateFields = async (
+    pageFields: Array<IPageField>,
+    overrideBoolean: boolean,
```

Comment:
Had to add this to workaround the override

Reply:
`overrideBoolean` isn't used as far as I can see

Reply:
Yes, but if I omit it there is an error as I cannot exclude it from the method override

Reply:
Sorry, not paying enough attention.